README.md

```diff
@@ -6,7 +6,8 @@
 
 * Clone this repo to your local device
 * `cd` into the repo directory
-* Run `npm install` (only for the first time that you run the app)
+* Run `npm install` (only for the first time you run the app)
+* Run `npm install @craco/craco --save` (only for the first time you run the app)
```

Comment:
@marisahoenig Contributors won't need to install craco because we added it to the package.json. When a contributor pulls down and runs `npm install` it will install craco for them.